CorsixTH/Lua/entities/object.lua

```diff
@@ -721,7 +721,7 @@ end
 
 function Object:afterLoad(old, new)
   if old < 52 then
-    self.hospital = self.world:getLocalPlayerHospital()
+    self.hospital = TheApp.ui.hospital
```

Comment:
This afterload is very old. Is it worth changing?

Reply:
(there's no harm changing these unless the ui didn't know of hospital at that point of history)